docs/sources/community/maintaining/release/concepts/version.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Version
+description: Version
+---
+# Version
+
+Grafana Loki uses Semantic Versioning. The next version can be determined
+by looking at the current version and incrementing it. Once exception can be when the release is new `major` version (e.g: after 2.9.1 we decide to go 3.0.0 instead of 2.9.2)
```

Comment:
```suggestion
by looking at the current version and incrementing it. When the release is a new `major` version (e.g: after 2.9.1 we decide to go 3.0.0 instead of 2.9.2) this should not be done.
```